practice/031/ex.ml

```diff
@@ -0,0 +1,33 @@
+open OUnit2
+
+module type Testable = sig
+  val is_prime : int -> bool
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let rec flag_primes n = function
+  | [] -> []
+  | p :: u ->
+    if n = p then 
```

Comment:
```suggestion
    if n = p then
```

Reply:
Thank you @cuihtlauac for your feedback and guidance.